src/makeRenderMethods.ts

```diff
@@ -14,9 +14,10 @@ export default function makeRenderMethods({
   // Update whole document content with SSR html
   // @NOTE we have to preserve document.body element identity
   // to not break @testing-library global "screen" object
+
+  // @TODO: Update html element attributes
```

Comment:
Maybe there's an obvious solution, I've never felt confident with DOM API's :) . Here we update the existing DOM with the rendered `html` string. `<html/>` is the root element for both.

Since we update DOM elements by injecting `html` string into root DOM `<html>` via `innerHtml`, this means that we currently ignore the `<html>` attributes rendered in `html` string.

If we cannot find a DOM-native solution, we might introduce a HTML parser to extract what we need from `html` string and apply it to the DOM.